docs/rules/no-console.md

```diff
@@ -45,6 +45,17 @@ console.warn("Log a warn level message.");
 console.error("Log an error level message.");
 ```
 
+`disallowGlobals`: A list of global objects not to be used in addition to any `console` references.
```

Comment:
Can you update this to our new format for rule options? See http://eslint.org/docs/rules/comma-dangle as an example. Each option should have its own heading.